.github/workflows/linters.yml

```diff
@@ -0,0 +1,39 @@
+name: Linters
```

Comment:
are you planning on killing off Travis? I'm not for or against just curious

Reply:
It's ideal to try to keep as much in-repo as possible.  

Seems GitHub has these Actions, it's possible to get away with not requiring Travis on the project.

I found Travis to build slower than Github actions, when testing it earlier (with all the force pushes)